AndroidSignalRJavaClientSample/app/src/main/java/com/example/mikaelmengistu/androidsignalrjavaclientsample/MainActivity.java

```diff
@@ -0,0 +1,66 @@
+package com.example.mikaelmengistu.androidsignalrjavaclientsample;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ListView;
+import android.widget.TextView;
+
+import com.example.mikaelmengistu.androidsignalrjavaclientsample.R;
+import com.microsoft.aspnet.signalr.HubConnection;
+import com.microsoft.aspnet.signalr.LogLevel;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        HubConnection hubConnection = new HubConnection("YOUR URL HERE", LogLevel.Debug);
```

Comment:
Use HubConnectionBuilder

Reply:
`HubConnectionBuilder` isn't available in the current published build. It will be in the final preview1 build though

Reply:
As soon as we have a build I can publish it and we can update this.